src/start.rs

```diff
@@ -21,7 +22,7 @@ pub struct Start {
     sub_command: SubCommand,
 }
 
-pub fn start(cmd: &Start, disable_exec: bool) -> Result<i32, &'static str> {
+pub fn start(cmd: &Start) -> Result<i32, &'static str> {
```

Comment:
I realized I can just use the `cfg!` to determine if I can disable exec instead of having this take an argument.